features/specs/gezag-minderjarige/niet-ouder-gezag.feature

```diff
@@ -0,0 +1,218 @@
+#language: nl
+
+Functionaliteit: in verschillende situaties heeft een niet ouder (ook) het gezag over een minderjarige
+
+  Een `niet ouder` is iemand die gezag heeft over een minderjarige maar niet de ouder is.
+  Dat wil zeggen, de persoon is:
+  - niet de moeder uit wie het kind geboren is
+  - niet iemand die het kind erkend heeft
+  - niet iemand die het kind geadopteerd heeft
+  Een `niet ouder` kan een vader zijn, maar ook een oma, opa, peetoom of andere relatie.
+
+  De gezag relatie voor een `niet ouder` met een minderjarige onstaat vanuit rechtswege of vanuit een gerechtelijke uitspraak.
+
+  Bij rechtswege ontstaat de gezagsrelatie wanneer de `niet ouder` gehuwd is met de ouder van de minderjarige.
+
+  Bij een gerechtelijk uitspraak kan worden gesteld dat de ouder het gezag gezamelijk met een `niet ouder` (derde) het gezag uitoefend.
+  Ook kan een gerechtelijke uitspraak gedaan worden die stelt dat 1 of meerdere `niet ouder(s)` (derden) het gezag uitoefenen.
+
+  Achtergrond:
+    Gegeven de persoon 'Annabel' met burgerservicenummer '000000012'
+    * is meerderjarig, niet overleden en staat niet onder curatele
+    En de persoon 'Rupert' met burgerservicenummer '000000024'
+    * is meerderjarig, niet overleden en staat niet onder curatele
+    En de persoon 'Yves' met burgerservicenummer '000000036'
+    * is minderjarig
+    * is in Nederland geboren
+    * is ingeschreven in de BRP
+
+  Regel: de niet ouder heeft vanuit rechtswege gezag over de minderjarige
+
+    Abstract Scenario: gezamelijk gezag van rechtswege door huwelijk met de ouder er is sprake van GezamelijkGezag
+      Gegeven persoon 'Yves'
+      * heeft 'Annabel' als ouder 1
+      En 'Annabel' en 'Rupert' zijn met elkaar gehuwd
```

Comment:
dit is alleen waar wanneer 'Annabel' en 'Rupert' al ten tijde van de geboorte van 'Yves' gehuwd waren (Yves is staande huwelijk geboren). Dat hebben we toevallig zo in de automation gemaakt, maar blijkt niet uit de tekst.